MediaPlayer/app/src/main/java/github/nisrulz/example/mediaplayer/MainActivity.java

```diff
@@ -0,0 +1,64 @@
+package github.nisrulz.example.mediaplayer;
+
+import android.media.AudioManager;
+import android.media.MediaPlayer;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+
+// Contributed by Sergio Morales (@Fireblend)
```

Comment:
Since git system attributes the author to each line of code, adding this is not required. A simple git blame enlists all the information related to the commit/line of code.

Reply:
Got it. Will remove line.